BlazorUI/Data/Caterogy.cs

```diff
@@ -0,0 +1,8 @@
+namespace BlazorUI.Data
+{
+    public class Caterogy
+    {
+        public string Name { get; set; }
+        public bool IsSelected { get; set; }
+    }
+}
```

Comment:
Codacy found an issue: [Add a new line at the end of the file 'Caterogy.cs'.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5088867)